chain-impl-mockchain/src/vote/manager.rs

```diff
@@ -347,7 +347,7 @@ impl ProposalManager {
             result.add_vote(Choice::new(u8::try_from(choice).unwrap()), weight)?;
         }
 
-        if self.check((*total_stake).into(), governance, &result) {
+        if self.check(Stake(encrypted_tally.max_stake), governance, &result) {
```

Comment:
I think it was a mistake also in the previous implementation, this should be the total available voting power, not the one that participated in the election (which can be obtained just by looking at the results)